src/navigation/loggedNav.jsx

```diff
@@ -37,6 +37,33 @@ export default class LoggedNav extends Component {
           }}
         />
 
+        <Tab.Screen
+          name="OnProductDetail"
+          component={OnProductDetail}
+          options={{
+            headerShown: true,
+            headerBackVisible: false,
+            tabBarButton: () => null,
+            headerLeft: (props) => (
+              <FontAwesome5
+                onPress={() => this.props.navigation.navigate("UserProfile")}
+                name="user-circle"
+                style={{ marginLeft: 15 }}
```

Comment:
The style can call it in an external file.

Reply:
Done